src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -101,122 +85,27 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       } recover { case e: Throwable => RequestCompleteWithErrorReport(InternalServerError,
         "Unexpected error updating NIH link", e) }
       // Complete syncWhitelist and ignore as neither success nor failure are useful to the client
-      syncWhiteListResult onComplete {
-        case _ => profileResponse pipeTo parent
-      }
+      syncWhiteListResult pipeTo sender
```

Comment:
I think you changed the behavior here. The old behavior was do syncWhiteListResult then whatever the response pipe back profileResponse. I think the right way to do this is convert syncWhiteListResult from ```Future[x]``` to ```Future[Try[x]]``` (see FutureSupport.toFutureTry in rawls). Then ```syncWhiteListResultTry flatMap { _ => profileResponse } pipeTo sender